medusa/classes.py

```diff
@@ -186,12 +186,11 @@ def add_result_to_cache(self, cache):
 
     def create_episode_object(self):
         """Use this result to create an episode segment out of it."""
-        if self.actual_season and self.actual_episodes and self.show:
-            self.episodes = [self.show.get_episode(self.actual_season, ep) for ep in self.actual_episodes]
-        else:
-            log.exception('Trying to create an episode object, without the proper ingredients. '
-                          'Missing season: {season}, episodes: {episodes} or the show object: {show}',
-                          {'season': self.actual_season, 'episodes': self.actual_episodes, 'show': self.show})
+        if self.show and self.actual_season:
+            if self.actual_episodes:
+                self.episodes = [self.show.get_episode(self.actual_season, ep) for ep in self.actual_episodes]
+            else:
+                self.episodes = self.show.get_all_episodes(self.actual_season)
```

Comment:
Yeah I checked. This is used later to verify if it is a season pack. So for now we also can't change this.